web/src/styles/wiki.css

```
@@ -124,17 +124,30 @@
 
 /* ─── Layout: persistent page-tree sidebar + content (+ article right rail) ─── */
 .wiki-layout {
+  /* Either flanking panel can fold to a thin rail (--wk-panel-rail) so the
+     reading column reclaims the width. The active column widths live in
+     these custom props; the data-*-collapsed attributes below swap them, and
+     the grid templates simply consume them. */
+  --wk-panel-rail: 44px;
+  --wk-col-left: var(--wk-sidebar-left);
+  --wk-col-right: var(--wk-sidebar-right);
   display: grid;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)